clearnode/app_session_service.go

```diff
@@ -121,6 +163,13 @@ func (d *DepositUpdater) Update(ctx context.Context, tx *gorm.DB) (UpdateResult,
 
 	d.appSession.Version++
 
+	// Update session key usage for any session keys that were actually used
+	for sessionKey := range sessionKeysUsed {
+		if err := UpdateSessionKeyUsage(tx, sessionKey); err != nil {
```

Comment:
Under the hood the main purpose is updating used allowance, maybe later we can redesign it, because it seems like we do unnecessary queries